numpyro/hmc_util.py

```diff
@@ -590,6 +595,16 @@ def initialize_model(rng, model, *model_args, **model_kwargs):
     model_trace = trace(model).get_trace(*model_args, **model_kwargs)
```

Comment:
I think that we need to split rng into key and subkey, where key is used to seed model, and the other one to be used in `uniform` strategy.

Reply:
Does it matter? The random values from the seeded model aren't used (only the shapes are used), and we split the seed in the uniform initialization anyways.

Reply:
Kaka, you are absolutely right!